repaygithub/ui-tools
# Incorporate scripts createui

How to test:

- Pull this branch 
- Run create-repay-ui and check it runs ok and there is no  webpack.config file generated in the target folder
